### PR TITLE
Add roycaihw to reviewers and update integration test README

### PR DESCRIPTION
**Pull request by roycaihw**
Add @roycaihw to reviewers in OWNER file per request from @apelisse. 

Also update `test/integration/README.md` to mention that the checked-in `pkg/generated/openapi_generated.go` file [should not be updated](https://github.com/kubernetes/kube-openapi/pull/104#issuecomment-424858284). 

/cc @apelisse

### CODE REVIEW

**Comment by apelisse** [comment]
I have the feeling that you should be an approver somewhere in there, maybe create a new OWNERS file?

**Comment by roycaihw** [comment]
@apelisse Added a hierarchical `pkg/generators/rules/OWNERS` file. It should likely be enough for me to approve when people are updating existing rules. 

However when new rules are being added, I'd expect `pkg/generators/api_linter.go` (one level higher) and files under `test/integration/testdata/` to also be updated (e.g. https://github.com/kubernetes/kube-openapi/pull/106), in which case I need to bug other approvers. 

Also still adding me to top-level reviewers if there is no objection.

File: pkg/generators/rules/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+reviewers:
+- roycaihw
+approvers:
+- roycaihw
```
Comment (mbohlool) [inline pkg/generators/rules/OWNERS]: we should have more than one approver to make sure one person is not bottleneck. You want to go on vacations, right? :) I suggest at least at me and lavalamp here.

Comment (mbohlool) [reply pkg/generators/rules/OWNERS]: my assumption is the ONWERS here overwrites the top level one, if that is not the case,  ignore my comment.

Comment (apelisse) [reply pkg/generators/rules/OWNERS]: it automatically inherits ownership from parent OWNERS. We're good!

**Pull request merged**